image/app/src/create/config.rs

```diff
@@ -13,20 +13,24 @@ Abstract:
 --*/
 
 use anyhow::Context;
-use caliptra_image_types::{VENDOR_ECC_KEY_COUNT, VENDOR_LMS_KEY_COUNT};
+use caliptra_image_types::{VENDOR_ECC_MAX_KEY_COUNT, VENDOR_LMS_MAX_KEY_COUNT};
 use serde_derive::{Deserialize, Serialize};
 use std::path::PathBuf;
 
 /// Vendor Key Configuration
 #[derive(Default, Serialize, Deserialize)]
 pub(crate) struct VendorKeyConfig {
-    pub ecc_pub_keys: [String; VENDOR_ECC_KEY_COUNT as usize],
+    pub ecc_key_count: u32,
 
-    pub lms_pub_keys: [String; VENDOR_LMS_KEY_COUNT as usize],
+    pub ecc_pub_keys: [String; VENDOR_ECC_MAX_KEY_COUNT as usize],
```

Comment:
It seems weird to me to have a fixed-length array for a variable-length structure. Maybe we should switch to a `Vec<String>` so that we don't get extra stuff in the serialized TOML?

Reply:
Updated to a vector and removed the key count fields from the keys.toml file.